design/status-design-update.md

```diff
@@ -0,0 +1,1166 @@
+          port: 80
+      status:
+        currentStatus: Valid
+        description: Valid route
+      includePath: "include-root:root.bar.com,service2"
+  proxyGraph:
+  - fqdn: root.bar.com
+    includes:
+      name: www
+      namespace: default
+      conditions:
+      - prefix: /service2
+      status:
+        condition: Valid
+        description: Valid include
+      routes:
+        - conditions:
+          - prefix: /service2
+          services:
+            - name: s2
+              port: 80
+          status:
+            currentStatus: Valid
+            description: Valid route
+        - conditions:
+          - prefix: /service2/blog
+          services:
+            - name: blog
+              port: 80
+          status:
+            currentStatus: Valid
+            description: Valid route
+  appliedConditions:
+  - fqdn: root.bar.com
+    includes:
+      - name: www
+        namespace: default
+        conditions:
+        - prefix: /service2
+```
+
+### Single HTTPProxy included by two roots
+
+This example illustrates the reason for namespacing the `proxyGraph` and `appliedConditions` fields.
+
+```yaml
+---
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: multiple-root
+  namespace: default
+spec:
+  virtualhost:
+    fqdn: bar.com
+  includes:
+  - name: main
+    namespace: default
+    conditions:
+      - prefix: /
+status:
+  currentStatus: valid
+  description: valid HTTPProxy
+  includes:
+    - name: main
+      namespace: default
+      conditions:
+      - prefix: /
+      status:
+        currentStatus: Valid
+        description: Valid include
+  routes: {}
+  proxyGraph:
+  - fqdn: bar.com
+    includes:
+    - name: main
+      namespace: default
+      conditions:
+      - prefix: /
+      status:
+        condition: Valid
+        description: Valid include
+  appliedConditions: {}
+---
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: multiple-root-www
+  namespace: default
+spec:
+  virtualhost:
+    fqdn: www.bar.com
+  includes:
+  - name: main
+    namespace: default
+    conditions:
+      - prefix: /
+status:
+  currentStatus: valid
+  description: valid HTTPProxy
+  includes:
+  - name: main
+    namespace: default
+    conditions:
+      - prefix: /
+    status:
+      currentStatus: Valid
+      description: Valid include
+  routes: {}
+  proxyGraph:
+  - fqdn: www.bar.com
+    includes:
+    - name: main
+      namespace: default
+      conditions:
+      - prefix: /
+      status:
+        condition: Valid
+        description: Valid include
+  appliedConditions: {}
+
+---
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: main
+  namespace: default
+spec:
+  routes:
+  - services:
+    - name: s2
+      port: 80
+status:
+  currentStatus: valid
+  description: valid HTTPProxy
+  includes: {}
+  routes:
+    - fqdn: bar.com
+      conditions:
+      - prefix: /
+      services:
+      - name: s2
+        port: 80
```

Comment:
I know this was mentioned, but would be nice to have a way to represent this without duplicating over. I need to think about this some more to provide an alternative.